client-src/elements/form-definition.js

```diff
@@ -700,18 +701,79 @@ const BLINK_GENERIC_QUESTIONNAIRE = (
 
 const ADOPTION_GENERIC_QUESTIONNAIRE = (
-  'To request a review, use the "Request review" button ' +
-  'above.'
+  `To request a review, use the "Request review" button above.`
 );
 
 const PRIVACY_GENERIC_QUESTIONNAIRE = (
-  'To request a review, use the "Request review" button ' +
-  'above.'
+  `To request a review, use the "Request review" button above.`
 );
 
 const SECURITY_GENERIC_QUESTIONNAIRE = (
-  'To request a review, use the "Request review" button ' +
-  'above.'
+  `To request a review, use the "Request review" button above.`
+);
+
+const ENTERPRISE_SHIP_QUESTIONNAIRE = (
+  html`<b>(1) Does this launch include a breaking change?</b>
+Does this launch remove or modify existing behavior or does it interrupt an existing user flow? (e.g. removing or restricting an API, or significant UI change). Answer with one of the following options, and/or describe anything you're unsure about:
+<ul>
+<li>No. There's no change visible to users, developers, or IT admins (e.g. internal refactoring)
+<li>No. This launch is strictly additive functionality
+<li>Yes. Something that exists is changing or being removed (even if usage is very small)
+<li>I don't know. Enterprise reviewers, please help me decide. The relevant information is: ______
+</ul>
+<b>(2) Is there any other reason you expect that enterprises will care about this launch?</b>
+(e.g. they may perceive a risk of data leaks if the browser is uploading new information, or it may be a surprise to employees resulting in them calling their help desk). Answer with one of the following options, and/or describe anything you're unsure about:
+<ul>
+<li>No. Enterprises won't care about this
+<li>Yes. They'll probably care because ______
+<li>I don't know. Enterprise reviewers, please help me decide. The relevant information is: ______
+</ul>
+<b>(3) Does your launch have an enterprise policy to control it, and will it be available when this rolls out to stable (even to 1%)?</b>
+Only required if you answered Yes to either of the first 2 questions. Answer with one of the following options, and/or describe anything you're unsure about:
+<ul>
+<li>Yes. It's called ______. It will be a permanent policy, and it will be available when stable rollout starts
+<li>Yes. It's called ______. This is a temporary transition period, so the policy will stop working on milestone ___. It will be available when stable rollout starts
+<li>No. A policy is infeasible because ______ (e.g. this launch is a change in how we compile Chrome)
+<li>No. A policy isn't necessary because ______ (e.g. there's a better method of control available to admins)
+</ul>
+<b>(4) Provide a brief title and description of this launch, which can be shared with enterprises.</b>
+Only required if you answered Yes to either of the first 2 questions. This may be added to browser release notes. Where applicable, explain the benefit to users, and describe the policy to control it.
+`
+);
+
+const DEBUGGABILITY_ORIGIN_TRIAL_QUESTIONNAIRE = (
```

Comment:
Does it need html prefix?

Reply:
Nope.  For that one, I am using a plain string, when then gets auto-linked, and it has white-space:pre-wrap, so blank lines organize the text into paragraphs.